lib-injection/run-lib-injection.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
where will this be used?

Reply:
this is used to run the library injection tests, unfortunately we couldn't use `run.sh` because it's currently being used in the node/java/python CIs to run the tests